### ISSUE TITLE
ETL autoridades municipales

### ISSUE THREAD

**Issue by aVolpe** [InstIDEA/controlciudadano#96]
Importar datos de autoridades electas de las últimas elecciones municipales y generales disponibles en el portal de datos abiertos del Tribunal Superior de Justicia electoral. Desarrollo de funcionalidades de búsqueda y descarga de datos con los datos importados

**Comment by aVolpe** [comment]
@Luxter77 esto al final no fué un ETL verdad? sino que se importaron de a 1 y en futuras elecciones se debera importar la información a mano?

**Comment by Luxter77** [comment]
Correcto, su página para obtener esos datos es demasiado compleja y desorganizada como para poder automatizar el proceso

**Comment by aVolpe** [comment]
@Luxter77 podes entonces describir los pasos que hiciste en este issue así queda como referencia para la siguiente vez que debemos importar datos?
